driver/configurations/cache.cmake

```
@@ -392,9 +392,11 @@ else()
       OUTPUT_VARIABLE DASHBOARD_LSB_RELEASE_DESCRIPTION_OUTPUT_VARIABLE
       OUTPUT_STRIP_TRAILING_WHITESPACE
     )
+    # NOTE: Focal reports `Ubuntu 20.04.4 LTS` whereas Jammy reports
+    # `Ubuntu 22.04 LTS`, the third number must be optional.
```

```suggestion
    # `Ubuntu 22.04 LTS`; the third number must be optional.
```

...though this makes me wonder if Focal reported `20.04` at launch, or if Jammy might eventually report `22.04.1`, `22.04.2`, and so forth.

---

Comment changed, irrelevant now.